examples/EthSwapNGo/react/src/utils/consts.ts

```diff
@@ -208,21 +208,21 @@ export const ETH_BRIDGE_ADDRESS = getAddress(
   CLUSTER === "mainnet"
     ? "0x98f3c9e6E3fAce36bAAd05FE09d375Ef1464288B"
     : CLUSTER === "testnet"
-    ? "0x706abc4E45D419950511e474C7B9Ed348A4a716c"
+    ? "0x4370eE0D87282A329595D98c285Ac4A3725490fb"
```

Comment:
EthSwapNGo is an DEX example?

Reply:
Yes, it can swap ETH and other chain tokens.

Reply:
great! we could reuse this UI to build our own DEX